batch/batch/batch.py

```diff
@@ -341,17 +350,16 @@ def _keep_alive_container(self):  # pylint: disable=R0201
             pod_status, err = await app['k8s'].read_pod_status(self._pod_name, pretty=True)
             if err is not None:
                 traceback.print_tb(err.__traceback__)
-                log.info(f'ignoring: could not get pod status for {self.id} '
-                         f'due to {err}')
+                self.log_info(
+                    f'ignoring: could not get pod status due to {err}')
```

Comment:
one line

Reply:
also 81 characters, but I made it one line